--- .tmux/kube-tmux/kube.tmux.orig	2019-07-21 22:02:27.410000000 +0200
+++ .tmux/kube-tmux/kube.tmux	2019-07-21 21:57:43.040000000 +0200
@@ -134,18 +134,18 @@
 
   # Symbol
   if [[ "${KUBE_TMUX_SYMBOL_ENABLE}" == true ]]; then
-    KUBE_TMUX+="#[fg=blue]$(_kube_tmux_symbol)#[fg=colour${1}]"
+    KUBE_TMUX+="#[fg=${KUBE_TMUX_SYMBOL_COLOR}]$(_kube_tmux_symbol)"
   fi
 
   # Context
-  KUBE_TMUX+="#[fg=${2}]${KUBE_TMUX_CONTEXT}"
+  KUBE_TMUX+="#[fg=${KUBE_TMUX_CTX_COLOR}]${KUBE_TMUX_CONTEXT}"
 
   # Namespace
   if [[ "${KUBE_TMUX_NS_ENABLE}" == true ]]; then
     if [[ -n "${KUBE_TMUX_DIVIDER}" ]]; then
       KUBE_TMUX+="#[fg=colour250]${KUBE_TMUX_DIVIDER}"
     fi
-    KUBE_TMUX+="#[fg=${3}]${KUBE_TMUX_NAMESPACE}"
+    KUBE_TMUX+="#[fg=${KUBE_TMUX_NS_COLOR}]${KUBE_TMUX_NAMESPACE}"
   fi
 
   echo "${KUBE_TMUX}"
